evap/static/ts/src/infobox.ts

```diff
@@ -6,6 +6,7 @@ export class InfoboxLogic {
     private readonly infobox: HTMLDivElement;
     private readonly closeButton: HTMLButtonElement;
     private readonly storageKey: string;
+    private timeout?: NodeJS.Timeout;
```

Comment:
This seems odd, we are not targetting node, so the inferred type should be a browser timeout, not a nodejs one. Do we have typescript misconfigured somewhere? 🤔

Reply:
okay: by default, tsc imports all `@types/*` into the global namespace. As we have `@types/node` because of jest, this works here.

If I excluded the jest types (and thereby node's types) this compiles with `number` but compilation of the ts tests fail, see https://github.com/e-valuation/EvaP/pull/2237/commits/e501844e1ecb1b4280a9d91684e052c31b38a6fa

Reply:
Ah, okay. According to [this](https://stackoverflow.com/a/51040768), using `window.setTimeout` (or the returntype trick they use) could work - if not, we can come back to this later

Reply:
now it works! Adding it to `tsconfig.compile.json` should work!